libs/design-system/app-logos/src/lib/app-logos.component.scss

```diff
@@ -47,6 +47,17 @@
       letter-spacing: var(--sys-label-medium-tracking);
       color: var(--sys-on-secondary-fixed);
     }
+
+    // Temporary
+    .beta-description {
```

Comment:
I wonder if there is a better way to do this or if we should ask Libby not to have 'Beta' as a description? There is a major risk that we forget to remove this when changing to a proper description.

Reply:
I updated the header in #778 to use the software status indicator, so this won't be needed anymore after that is merged in.